src/WPBT/WPBT_Core.php

```diff
@@ -223,4 +229,39 @@ public function add_admin_page( $tab, $action ) {
 		</div>
 		<?php
 	}
+
+	/**
+	 * Get the next version the site will be updated to.
+	 *
+	 * @since 2.2.0
+	 *
+	 * @param string $preferred_version The preferred version.
+	 * @return string
+	 */
+	function get_next_version( $preferred_version ) {
+		if ( ! ( 0 === strpos( static::$options['stream'], 'beta-rc' ) ||
+				! preg_match( '/alpha|beta|RC/', get_bloginfo( 'version' ) ) ) ) {
+			// site is not running a development version or not on a beta/RC stream.
+			// So use the preferred version.
+			return $preferred_version;
+		}
+
+		$next_version = $this->wp_beta_tester->beta_rc->get_found_version();
+		if ( $next_version ) {
+			// the next beta/RC package was found, return that version.
+			return $next_version;
+		}
+
+		// the next beta/RC package was not found.
+		$next_version = $this->wp_beta_tester->beta_rc->next_package_versions();
+		if ( count( $next_version ) === 1 ) {
```

Comment:
Yoda, you must. 😉

Reply:
These are not the kinds of conditions that you're looking for.

since neither side of the comparison operator is a variable, the order doesn't matter.  To quote the [handbook](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions):

> When doing logical comparisons involving variables, always put the variable on the right side and put constants, literals, or function calls on the left side. If neither side is a variable, the order is not important. (In computer science terms, in comparisons always try to put l-values on the right and r-values on the left.)